lib/desktop_screens_new/notification/sending_file_card.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:io';
+
+import 'package:atsign_atmosphere_pro/data_models/file_transfer.dart';
+import 'package:atsign_atmosphere_pro/view_models/file_transfer_provider.dart';
+import 'package:flutter/material.dart';
+
+class SendingFileCard extends StatelessWidget {
+  final FLUSHBAR_STATUS? flushbarStatus;
+  final FileTransfer? fileTransfer;
+  const SendingFileCard({
+    Key? key,
+    this.flushbarStatus,
+    this.fileTransfer,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return flushbarStatus == FLUSHBAR_STATUS.SENDING && fileTransfer != null
+        ? Container(
+            width: double.infinity,
+            padding: EdgeInsets.symmetric(horizontal: 20, vertical: 10),
+            decoration: BoxDecoration(
+              color: Color(0xFFF4F4F4),
+              borderRadius: BorderRadius.circular(10),
+            ),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.start,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                RichText(
+                  text: TextSpan(
+                    children: [
+                      TextSpan(
+                        text: 'sending',
+                        style: TextStyle(
+                            fontSize: 11,
+                            fontWeight: FontWeight.bold,
+                            color: Colors.black),
+                      ),
+                      TextSpan(
+                        text: '${fileTransfer!.files?.length ?? 0} files',
+                        style: TextStyle(fontSize: 11, color: Colors.black),
+                      )
+                    ],
+                  ),
+                ),
+                SizedBox(
+                  width: (Platform.isAndroid || Platform.isIOS) ? 200 : 230,
+                  child: Text(
+                    getAtsignCountFromFileTransfer(fileTransfer!),
+                    style: TextStyle(fontSize: 9, fontWeight: FontWeight.bold),
+                  ),
+                ),
+                SizedBox(height: 5),
+                LinearProgressIndicator(
+                  minHeight: 10,
+                  borderRadius: BorderRadius.circular(5),
```

Comment:
@sachins-geekyants  `Error: No named parameter with the name 'borderRadius'`

Reply:
I can run it
<img width="544" alt="Screenshot 2023-10-26 at 4 44 52 PM" src="https://github.com/atsign-foundation/atmosphere_pro/assets/74917871/00ff677b-6956-48b1-abb0-8090dfa8de7b">